tests/sentry/incidents/models/test_alert_rule.py

```diff
@@ -215,6 +215,30 @@ def test_multi_project(self):
             AlertRule.objects.fetch_for_organization(self.organization, [project])
         )
 
+    def test_project_on_alert(self):
+        project = self.create_project()
+        alert_rule = self.create_alert_rule()
+        alert_rule.projects.add(project)
```

Comment:
You _should_ be able to add `projects` to the `create_alert_rule` now so you won't need to add this step!

until the migration PR gets in though, it will only be available for `activated` alert rules